public/app/plugins/datasource/influxdb/components/editor/config-v2/AuthSettings.tsx

```diff
@@ -132,35 +142,33 @@ export const AuthSettings = (props: Props) => {
         </InlineField>
       </Box>
 
-      {/* Collapsible settings body */}
       {authenticationSettingsIsOpen && (
         <Box paddingLeft={1}>
-          {/* Authentication Method */}
```

Comment:
I see you've removed the comments here, but I think we should keep them for now until we break this component down even smaller, to make it more readable.